src/TodoRouter.js

```diff
@@ -0,0 +1,23 @@
+import React, { useContext } from 'react'
+import { BrowserRouter as Router, Route, Redirect } from 'react-router-dom'
+import AuthContext from './AuthContext'
+import TodoList from './TodoList'
+import Login from './login'
+
+const TodoRouter = () => {
+  const {token} = useContext(AuthContext)
```

Comment:
nghiên cứu cách viết Public route, Private route như này em nhé
https://dev.to/nilanth/how-to-create-public-and-private-routes-using-react-router-72m